spec/features/admin/admin_cannot_modify_users_spec.rb

```diff
@@ -1,3 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'Admin can modify their account data' do
+  scenario 'Admin can not modify user data' do
+    admin = create(:user, role: 1)
+    user = create(:user)
+
+    visit login_path
+
+    fill_in 'Username', with: admin.username
+    fill_in 'Password', with: admin.password
+
+    click_on 'Save Session'
+
+    expect(page).to have_content('Modify Account Data')
+
+    click_on 'Modify Account Data'
+    expect(current_path).to eq(edit_user_path(admin))
```

Comment:
This tests that 'Modify Account Data' goes to the right place, but it doesn't test that an admin couldn't modify a user if they were to try going straight to edit_user_path(user). It might be worth trying to visit that in the test and ensuring that they don't see field to edit the user data.